aerogear/aerogear.org
# fix broken link

## Motivation
https://github.com/aerogear/aerogear.org/issues/776

## What
All instances of the link updated

## Why
To fix the broken links

## Verification Steps
Build it and check the links
The main one to check is here: https://aerogear.org/community/#contact as the other pages aren't in use currently. but I'm updating them to be thorough.
 
## Checklist:


@camilamacedo86 time to review this one? I noticed the broken link while I was reviewing [this pr](https://github.com/aerogear/aerogear.org/pull/773)
